velox/dwio/common/tests/TypeTests.cpp

```diff
@@ -29,6 +29,19 @@ using namespace facebook::velox::dwio::common::typeutils;
 using facebook::velox::type::fbhive::HiveTypeParser;
 using facebook::velox::type::fbhive::HiveTypeSerializer;
 
+void compareSchema(
```

Comment:
I think it's better to rename it to `assertEqualTypeWithId`, because `compareSchema` doesn't tell the reader what they should expect from this function.